light_node/src/main.rs

```diff
@@ -233,10 +233,19 @@ fn block_on_actors(
             .num_of_peers_for_bootstrap_threshold(),
     );
 
+    let mut actor_system_cfg = riker::load_config();
+    if env.riker_threads > 0 {
+        let thread_count = env.riker_threads.min(num_cpus::get());
+        actor_system_cfg
+            .set("dispatcher.pool_size", thread_count as i64)
+            .unwrap();
```

Comment:
@binier 
please use `expect` with message instead of unwrap

Reply:
It rikers source code inside `riker::load_config()` unwrap is used for that. But ok will replace it with `expect`.